bin/kubectl-crossplane-stack-init

```diff
@@ -3,19 +3,28 @@
 set -e
 
 function usage {
-  echo "Usage: kubectl crossplane stack init STACK_NAME [DIRECTORY]" >&2
+  echo "Usage: kubectl crossplane stack init [-f] STACK_NAME [DIRECTORY]" >&2
   echo "" >&2
   echo "STACK_NAME is used as the image name for the stack, and any '/' characters" >&2
   echo "are converted to '-' characters when populating kubernetes resource field names." >&2
   echo "" >&2
   echo "DIRECTORY defaults to the current directory." >&2
+  echo "" >&2
+  echo "-f, --force force init when a previous 'stack init' is detected" >&2
```

Comment:
Could you put some sort of delimiter between the argument and the explanation so this line is easier to scan? For example: `-f, --force : force init . . .`

Reply:
I was trying to follow the format of a well-known command. In this case `ls`.

I think your last few requests highlight that I've been targeting the wrong tool, and for these requests to seem less arbitrary we should standardize on the most relevant tool: `kubectl`.

For example:

```
kubectl delete --help
```

```
$ kubectl command --help
Command does thing by thing thinger.

 ... length usage notes

 IMPORTANT: ... warnings and notes

Examples:
  # description of command
  kubectl ...arguments
  
  # .. more examples

Options:
  -O, --option=default: description
    ...

Usage:
  kubectl command ([-O ARGUMENT] | [-x X] | TYPE [(NAME | -l label | --all)]) [options]

Use "kubectl options" for a list of global command-line options (applies to all commands).
```